### DISCUSSION TITLE
Loop out and payment routing failures

### DISCUSSION THREAD

**Discussion by thibistaken** [Ride-The-Lightning/RTL#1390] [Troubleshooting Questions] [answered]
Hi @saubyk and team. I've tried about 25 loop outs on RTL but they keep failing instantly. Payments also fail often. Got 6 successful out of 40+. Any clue why that is? 

- I run BTCPay Server v1.13.1 and LND 0.17.4-beta. I [asked the BTCPay team](https://t.me/btcpayserver/73498) about a troubleshooting but no luck so far.
- I opened one 930k sats channel with a peer that is well ranked on mempool.space in terms of connectivity (top 20). 
- With that channel peer, I have outbound liquidity (~928k) and I'm trying to get inbound liquidity. 
- Every loop out attempt results in "failed" state directly after being "initiated" for a few seconds. No additional context.
- I also tried sending bitcoin to my own LN wallet (Phoenix) to move liquidity to the remote side of the channel but most of my payments fail with "no route" error. 
- I have 6 payments that succeeded over 40+ attempts, with only small payments of 1k, 2k, 5k and 10k sats. I currently have 18k of inbound liquidity. 

Of course, I'd like to balance the channel to get closer to a balance score of 1. Any idea what I could be doing wrong and what these errors are about? Could it be that my peer has not outbound  liquidity with their other channel peers so payments can't route? I read some [docs](https://lightning.engineering/loop/) about loop out and [forums](https://www.reddit.com/r/lightningnetwork/comments/krkzbs/lightning_loop_out_rarely_successfully/) about inbound liquidity but nothing about this error troubleshooting.

Thanks for your help!

**Comment by saubyk** [comment, marked as answer]
Hi @thibistaken 
If the loop-out is failing that quickly, the most likely reason can be a low routing fee cap, which might be resulting in the lightning payment not getting to the loop node. Loop is a high demand service on the network, which typically results in high routing fee rates charged by the routing nodes carrying the transaction traffic to the loop node.

I would recommend raising the `Max Off-chain Routing Fee (%)` value to a higher value than the default of 2%, and see if that helps.
![Screenshot 2024-05-12 at 3 33 27 PM](https://github.com/Ride-The-Lightning/RTL/assets/39208279/a0595745-5eaf-4903-b7dd-5755e0cb8067)

> I also tried sending bitcoin to my own LN wallet (Phoenix) to move liquidity to the remote side of the channel but most of my payments fail with "no route" error.
I have 6 payments that succeeded over 40+ attempts, with only small payments of 1k, 2k, 5k and 10k sats. I currently have 18k of inbound liquidity.

This is also concerning. Are you applying some sort of a routing fee cap, which may be limiting the transactions with larger amount?

For whatever reason your node is not able to send bigger transaction amounts. The simplest explanation which comes to mind is a less efficient graph. May be try connecting your node to more peers (need not open channels to them, although opening more channels also helps), so that your node sources graph information from multiple peers on the network.

Hope this helps

**Reply by thibistaken** [reply]
Thanks for your help @saubyk. 

> I would recommend raising the Max Off-chain Routing Fee (%) value to a higher value than the default of 2%, and see if that helps.

I just tried multiple loop out attempts up to 10% Max Off-chain Fee but it did not work. I tried with 250k, which is the minimum amount allowed. Anything else I should try? 

> Are you applying some sort of a routing fee cap, which may be limiting the transactions with larger amount?

AFAIK no though I am not sure how I would apply a routing fee cap.

I connected my node to extra peers, a total of 8 for now and my payments still fail... anything else I may be missing? 

Thanks again.

**Reply by saubyk** [reply]
Another thing to experiment with is to open a channel with another peer and see if that improves the situation

**Reply by thibistaken** [reply]
Yes, I am trying that route. No luck so far but will update when I figure it out. Thanks
